exporters/jaeger/src/recordable.cc

```diff
@@ -19,10 +20,26 @@ void JaegerRecordable::PopulateAttribute(nostd::string_view key,
                                          const common::AttributeValue &value,
                                          std::vector<thrift::Tag> &tags)
 {
-  if (nostd::holds_alternative<int64_t>(value))
+  if (nostd::holds_alternative<int>(value))
```

Comment:
shouldn't we be comparing only for the supported types from here: https://github.com/open-telemetry/opentelemetry-cpp/blob/b1b1f6409a3a95e783f746920ba48006d8d32926/api/include/opentelemetry/common/attribute_value.h#L37

So, int, uint64_t are not valid attribute types.

Reply:
thanks, removed unsupported types